process-engine/iam
# 🔒 Verify parsed auth tokens

## Changes

1. Add test coverage evaluation
2. Robustify token parsing in IdentityService
3. Increase test coverage for IdentityService

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/396

PR: #12

## How to test the changes

Try passing all kinds of different values to the IdentityService and see if it behaves as you'd expect.
